Mail/Views/Switch User/AccountView.swift

```diff
@@ -27,13 +27,15 @@ import SwiftUI
 
 class AccountViewDelegate: DeleteAccountDelegate {
     @MainActor func didCompleteDeleteAccount() {
-        guard let account = AccountManager.instance.getCurrentAccount() else { return }
-        AccountManager.instance.removeTokenAndAccount(token: account.token)
-        if let nextAccount = AccountManager.instance.accounts.first {
-            AccountManager.instance.switchAccount(newAccount: nextAccount)
-            IKSnackBar.showSnackBar(message: "Account deleted")
+        Task {
```

Comment:
Can you explain me why we need to create a task

Reply:
We don't, at first TokenStore was an Actor and we had to await every call but I couldn't do this unless a lot of refactor was done so I stayed with our thread safe (but blocking) collections